pyccel/decorators.py

```diff
@@ -6,6 +6,7 @@
 """
 This module contains all the provided decorator methods.
 """
+import numpy
```

Comment:
I would prefer just importing what we need:
```suggestion
from numpy import array
```